live.senkins.SeventhHeavenWrapper.yml

```diff
@@ -25,7 +25,7 @@ finish-args:
   # Steam Flatpak
   - --filesystem=~/.var/app/com.valvesoftware.Steam:ro
   # Steam Deck - TODO we may also need xdg-data/Steam
-  - --filesystem=~/.steam:ro
+  - --filesystem=~/.local/share/Steam:ro
```

Comment:
This should be `xdg-data/Steam`.

Can remove the TODO as well, now we know this is needed

Reply:
Done